### ISSUE TITLE
[PL][QUEST][CATA] Delivery to Master Harris

### ISSUE THREAD

**Issue by misiumoto** [Atlantiss/BugTracker#6580]
Quest : Delivery to Master Harris
link : http://www.wowhead.com/quest=26719/delivery-to-master-harris#comments

Problem nie można oddać questa po wzięciu questa od Apprentice Fess
dowód 
http://imgur.com/8XQ3SG2

Jak powinno działać - powinno się oddać questa u tego npc , npc wyświetla puste okienko.

**Comment by user1011** [comment]
Bump.

**Comment by user1011** [comment]
In order to complete this quest you first need to finish Abercrombie's quest chain.
http://www.wowhead.com/npc=289/abercrombie

**Comment by reapertg** [comment]
Not true. I have done Abercombie's quest chain and killed Stitches in Darkshire, I am still unable to speak to  Oliver Harris to continue this quest chain

**Comment by TranceAccount** [comment]
Okay, but it still means it is bugged.

**Comment by reapertg** [comment]
agreed, it is bugged. thats what brought me here to this thread in the 1st place :p the NPC "Oliver Harris" will just not interact with you in any way.

**Comment by TranceAccount** [comment]
Have you tried completing every quest in the zone before Oliver Harris's chain?

**Comment by nekhen** [comment]
09.08.2017 Quest still does not work...

**Comment by nekhen** [comment]
10.08.2017 Quest completed.
